modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/data/provider/ContentDashboardDataProvider.java

```diff
@@ -261,6 +265,14 @@ private AssetVocabularyMetric _getAssetVocabularyMetric(
 			Collectors.toMap(Bucket::getKey, Bucket::getDocCount));
 	}
 
+	private String _getColor(String key) {
```

Comment:
For the shake of the POC, could you extract the color on the front-side using the category kay?

Reply:
OMG!!!! I can do it the same way in frontend side 🤯 Monday issues...I'm sorry.